deploy/grafana-dashboard/general_row.py

```diff
@@ -0,0 +1,112 @@
+from grafanalib.core import RowPanel, GridPos, Stat, TimeSeries, Heatmap, BarGauge, GAUGE_DISPLAY_MODE_GRADIENT
+from grafanalib.influxdb import InfluxDBTarget
+
+from color_mapping import grey_all_mapping
+from util import read_query
+
+requests_per_minute = TimeSeries(
+    title="Requests per minute",
+    dataSource='Poseidon',
+    targets=[InfluxDBTarget(query=read_query("requests-per-minute"))],
+    gridPos=GridPos(h=9, w=8, x=0, y=1),
+    scaleDistributionType="log",
+    extraJson=grey_all_mapping
+)
+
+request_latency = Heatmap(
+    title="Request Latency",
+    dataSource='Poseidon',
+    dataFormat="timeseries",
+    targets=[InfluxDBTarget(query=read_query("request-latency"))],
+    gridPos=GridPos(h=9, w=8, x=8, y=1),
+    extraJson={
+        "options": {},
+        "yAxis": {
+            "format": "ns"
+        }
+    }
+)
+
+service_time = TimeSeries(
+    title="Service time (99.9%)",
+    dataSource='Poseidon',
+    targets=[InfluxDBTarget(query=read_query("service-time"))],
+    gridPos=GridPos(h=9, w=8, x=16, y=1),
+    scaleDistributionType="log",
+    scaleDistributionLog=10,
+    unit="ns"
+)
+
+current_environment_count = Stat(
+    title="Current environment count",
+    dataSource='Poseidon',
+    targets=[InfluxDBTarget(query=read_query("current-environment-count"))],
+    gridPos=GridPos(h=6, w=8, x=0, y=10),
+    alignment='center'
+)
+
+currently_used_runners = Stat(
+    title="Currently used runners",
+    dataSource='Poseidon',
+    targets=[InfluxDBTarget(query=read_query("currently-used-runners"))],
+    gridPos=GridPos(h=6, w=8, x=8, y=10),
+    alignment="center"
+)
+
+number_of_executions = BarGauge(
+    title="Number of Executions",
+    dataSource="Poseidon",
+    targets=[InfluxDBTarget(query=read_query("number-of-executions"))],
+    gridPos=GridPos(h=6, w=8, x=16, y=10),
+    allValues=True,
+    displayMode=GAUGE_DISPLAY_MODE_GRADIENT,
```

Comment:
Shall we rotate the bars, similar to our current design?

```suggestion
    orientation=ORIENTATION_VERTICAL,
    displayMode=GAUGE_DISPLAY_MODE_BASIC,
```